docs/Installation/index.md

```diff
@@ -117,6 +117,11 @@ contour:
         domain: my-host.com
    ```
 
+    > Note: Since v0.1.9, elasticsearch is an optional component. By default, Paralus will install with Postgres. If you're doing a fresh install, below are the values.yaml file that you must pass during installation:
```

Comment:
```suggestion
    > Note: Since v0.1.9, elasticsearch is an optional component. By default, Paralus will use database ( Postgres ) as the auditlog storage component. If you're doing a fresh install, below are the values.yaml file that you must pass during installation:
```